rt_utils/rtstruct.py

```diff
@@ -12,9 +12,10 @@ class RTStruct:
     Wrapper class to facilitate appending and extracting ROI's within an RTStruct
     """
 
-    def __init__(self, series_data, ds: FileDataset, ROIGenerationAlgorithm=0):
+    def __init__(self, series_data, ds: FileDataset, ROIGenerationAlgorithm=0, warn_only: bool = False):
```

Comment:
Same thing as other comment, do we need this?

Reply:
Same as above [7798fde](https://github.com/qurit/rt-utils/pull/64/commits/7798fdede0c393419ca45c133a6b0f91a6c0d290)